test/test_khard.py

```diff
@@ -161,3 +161,14 @@ def test_find_email_addresses_finds_all_emails_with_other_headers_too(
                 username="mjones",
                 domain="machine.example")]
         self.assertEqual(expected, addrs)
+
+class TestSearchContacts(unittest.TestCase):
```

Comment:
Normally I name the test classes after the method they test and the individual test methods name describe *what* they test more precicely. Can you fix the names here please.